v1/suzu.json

```diff
@@ -24,6 +24,11 @@
     "confirm_model": {
       "title": "Confirm your model",
       "description": "Please double-check that your device is a Sony Xperia X (F5121 or F5122)."
+    },
+    "upgrade_android": {
+      "title": "Upgrade Android",
+      "description": "If the device is running Android, you might have to upgrade to Android 8.0.",
```

Comment:
I would be more strict: The device needs to be on Android 8.0 before flashing, there is no _might_ option for that. See also original wording: _"Make sure to upgrade your device to the latest stock Android 8.0 before continuing"_ - Also, the word upgrade is dangerous, we need to ask @fredldotme but shouldn´t it rather be _to reset your device...._ to emphasize we want exactly that version?

Reply:
I think the wording is okay. People coming from SailfishOS probably already have the right bootloader installed. Also I think that "resetting" sounds more like factory reset than upgrading to the right Android version.